src/services/item/plugins/publication/validation/index.ts

```diff
@@ -83,7 +84,12 @@ const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
       db.transaction(async (manager) => {
         const repositories = buildRepositories(manager);
         // get item and check permission
-        const item = await itemService.get(member, repositories, itemId, PermissionLevel.Admin);
+        const item = await folderItemService.get(
```

Comment:
Should we add a comment that lets us know we rely on querying the folderService to only allow folder items to be published ?